packages/address/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@shopify/address",
-  "version": "1.0.1",
+  "version": "1.0.2",
```

Comment:
@TheMallen, do we have to keep the version bump in a separate PR / commit?

Reply:
yes

Reply:
The `release` line tool will find all changed packages and let the person who runs it choose new versions